### DISCUSSION TITLE
Neste Component mount

### DISCUSSION THREAD

**Discussion by amaurycid** [livewire/livewire#4116] [1. Help]
Hi community!
If I nest a livewire component inside another livewire component, when the mount property executes in child, should I call parent mount property inside child..? like when we call __contruct() extending another class using parent::__construct()

**Comment by mattkibbler** [comment]
Hi @amaurycid,

The mount method is called automatically by LiveWire so you don't need to worry about calling it.

**Reply by amaurycid** [reply]
Got it. Thank you @mattkibbler

**Comment by joshhanley** [comment]
@amaurycid it depends on what you mean by nest, if it's nesting in blade, calling `<livewire:my-child-component>` inside your blade view of the parent component, then no, Livewire will call it for you.

But if you actually mean extending a component class like 

```php
class MyChildComponent extends MyParentComponent
```

Then yes, you will need to call `parent::mount()` if you have implemented mount in both your parent and child classes.

Hope this helps!

**Reply by amaurycid** [reply]
@joshhanley Thank you for your answer. Actually, I was concerned about extending a component class, but didn't make my question clear.
Cheers.

**Reply by joshhanley** [reply]
@amaurycid no worries! 🙂 Did the above answer work for you?
